daemons/controld/controld_messages.c

```diff
@@ -1202,6 +1201,8 @@ send_msg_via_ipc(xmlNode * msg, const char *sys)
 {
     pcmk__client_t *client_channel = pcmk__find_client_by_id(sys);
```

Comment:
pcmk__find_client_by_id is robust apart from expecting that libqb will handle the NULL.
are we doing it for the trace? Otherwise we could assign client_channel afterwards.

Reply:
do our coding conventions / personal likes say anything about

```
do {
  if // good case 1 {
    // good action 1
    break;
  } else if // common good case {
    if // good case 2 {
      // good action 2
      break;
    } else if ...
  }  
  // common error moaning
} while (0);
```